oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticBulkProcessorHandler.java

```diff
@@ -284,7 +297,7 @@ private RealTimeBulkProcessorHandler(@NotNull ElasticConnection elasticConnectio
         }
 
         @Override
-        public boolean close() {
+        public boolean close() throws Throwable {
```

Comment:
I'm against throwing Throwable, because that means you can't distinguish in the future between different root causes... For a discussion see https://stackoverflow.com/questions/12564428/is-throws-throwable-good-practice#:~:text=You%20should%20not%20throw%20Throwable,them%20in%20your%20method%20declaration.